qa-tests-backend/src/test/groovy/ReconciliationTest.groovy

```diff
@@ -154,7 +151,10 @@ class ReconciliationTest extends BaseSpecification {
                 log.info pod
             }
 
-            orchestrator.deleteAndWaitForDeploymentDeletion(sensorDeployment)
+            List<Pod> pods = orchestrator.getPodsByLabel("stackrox", ["app": "sensor"])
```

Comment:
We have a const for that in common package:
```suggestion
            List<Pod> pods = orchestrator.getPodsByLabel(Constants.STACKROX_NAMESPACE, ["app": "sensor"])
```

Reply:
Maybe you can fix all places in this file or we can do it as follow up

Reply:
I've changed all the appearances in this test 👍